### PR TITLE
jet-pilot 1.31.0 (new cask)

### PR DESCRIPTION
**Pull request by unxsist**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/j/jet-pilot.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  url "https://github.com/unxsist/jet-pilot/releases/download/v#{version}/JET.Pilot_#{version}_#{arch}.dmg",
+      verified: "github.com/unxsist/jet-pilot/"
+  name "JET Pilot"
+  desc "Open-source Kubernetes desktop client that focuses on speed and good looks"
+  homepage "https://www.jet-pilot.app/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (bevanjkay) [inline Casks/j/jet-pilot.rb:17]: If you don't use GitHub pre-releases, and no tags without a release, then the `livecheck` block can be ommitted.
The default `:git` strategy is inferred automatically from the `url` and is preferrable whenever possible as it does not require the use of the GitHub API.

Comment (unxsist) [reply Casks/j/jet-pilot.rb]: That makes sense, I've omitted the `livecheck` block!

File: Casks/j/jet-pilot.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  arch arm: "aarch64", intel: "x64"
+
+  version "1.31.0"
+  sha256  arm:   "77c787e74cfc415870a448d481caeb78523f5c803d08b0443dfb8cbb45b4bf86",
+          intel: "29fc8df5a2e133510e0c35cd49a05856045c75c917a0531f46d519e52a410cd7"
+
+  url "https://github.com/unxsist/jet-pilot/releases/download/v#{version}/JET.Pilot_#{version}_#{arch}.dmg",
+      verified: "github.com/unxsist/jet-pilot/"
+  name "JET Pilot"
+  desc "Open-source Kubernetes desktop client that focuses on speed and good looks"
```
Comment (bevanjkay) [inline Casks/j/jet-pilot.rb:11]:
```suggestion
  desc "Kubernetes desktop client"

**Review by bevanjkay — APPROVED** [review]

**Review by khipp — APPROVED** [review]
Thank you for your first contribution to `homebrew-cask`, @unxsist! 🎉

**Pull request merged**